src/actions/figure_menu_actions.ts

```diff
@@ -129,7 +129,33 @@ export function getCarouselMenuActions(
     getCutMenuItem(figureId, env),
     { ...getCopyAsImageMenuItem(figureId, env), isVisible: isChartSelected },
     { ...getDownloadChartMenuItem(figureId, env), isVisible: isChartSelected },
-    getDeleteMenuItem(figureId, onFigureDeleted, env),
+    {
+      id: "popout_chart",
+      name: _t("Pop out chart"),
+      sequence: 1,
+      icon: "o-spreadsheet-Icon.EXTERNAL",
+      execute: () => {
+        // get the selected carousel item
+        const selectedItem = env.model.getters.getSelectedCarouselItem(figureId);
+        if (!selectedItem || selectedItem.type !== "chart") {
+          return;
+        }
+        env.model.dispatch("POPOUT_CHART_FROM_CAROUSEL", {
+          carouselId: figureId,
+          chartId: selectedItem.chartId,
+          sheetId: env.model.getters.getActiveSheetId(),
+        });
+      },
+      isEnabled: isChartSelected,
+    },
+    {
+      ...getDeleteMenuItem(figureId, onFigureDeleted, env),
+      isVisible: (env) => {
+        // only visible if thre are no charts in the carousel
+        const carouselItems = env.model.getters.getCarousel(figureId)?.items;
+        return carouselItems?.length === 0 || !carouselItems?.some((item) => item.type === "chart");
```

Comment:
Shouldn't we have the "delete button" available when there is one or less item ? This would simplify the condition (even if it's a nitpick). And I don't know if it's an issue, but we can still delete the carousel with the "Delete" key. But I understand the hiding of this delete button is just an aesthetic thing.

Reply:
IMHO it's easier to understand that you don't have the delete button while there is a graph, but we can change that easily later if we want :)